src/presentation/pages/Home.vue

```diff
@@ -1,44 +1,68 @@
 <template>
-  <ThreeColsLayout>
+  <ThreeColsLayout data-dimensions="large">
+    <router-link
+      to="/new"
+    >
+      <Container>
+        <div :class="$style['container__create-new-note']">
+          <Row
+            :title="t('home.createNewNote.title')"
+            :subtitle="t('home.createNewNote.caption')"
+          >
+            <template #left>
+              <Hammer />
+            </template>
+            <template #right>
+              <Button
+                secondary
+                trailing-icon="ChevronRight"
+              >
+                {{ t('home.createNewNote.button') }}
+              </Button>
+            </template>
+          </Row>
+        </div>
+      </Container>
+    </router-link>
     <Heading
       :level="1"
-      :class="$style['home__heading']"
+      :class="$style['page-header']"
     >
       {{ $t('home.title') }}
     </Heading>
 
-    <div v-if="!user">
+    <div v-if="user === null">
       <p>{{ $t('home.authText') }}</p>
```

Comment:
could you include this message to the Row and Container as well? And probably add a Login button to the right
<img width="877" alt="image" src="https://github.com/codex-team/notes.web/assets/3684889/63778d24-2475-4ba5-bc89-54f1a9ca86d3">

Reply:
and I guess the Load More button should not be displayed in this case

Reply:
should we display create new note block when user is not authorized?

Reply:
no